### DISCUSSION TITLE
Questions about Subscriptions

### DISCUSSION THREAD

**Discussion by jackroc97** [tastytrade/tastytrade-sdk-python#9] [Q&A]
First off, I just want to say "great job!" to anyone who has contributed to this SDK.  I've been playing around with it for about an hour and have found it very user friendly and easy to pick up.  I do have a few questions about `Subscription`s:

1. Is there a reason that `Quote` does not have a `time` attribute like the other subscription types?  I would think that knowing the time of the quote would be important. 
2. What is the `price` attribute on `Greek`s?  Is this the "last" price (presumably, at the `time` of the `Greek`)?
3. Is there a hard limit on the number of symbols for a single `Subscription`?  In my messing around with it, I found that I've been limited to 200 symbols, but I'm not sure if/why this limit exists.

Thanks!

**Comment by kw** [comment]
This is a bit of a late answer, but the data from the subscriptions is actually from a third party called dxFeed. This library just passes that data through, so you may need to look into their [protocol docs](https://demo.dxfeed.com/dxlink-ws/debug/#/protocol) and [data model](https://kb.dxfeed.com/en/data-model/market-events/dxfeed-api-market-events.html) to learn more. You might also be able to ask someone directly at their github page (https://github.com/dxFeed) though at present, this library doesn't directly use theirs.

As for the questions, I can take a stab based on reading that doc...

1. Find the `QuoteEvent` message description in the doc, it says it has `askTime`, `bidTime`, and `eventTime` fields.
2. dxFeed's FAQ just says "Black-Scholes and Bjerksund-Stensland models are used to calculate Greeks." See also https://kb.dxfeed.com/en/data/calculated-data/dxliboptions-library.html
3. Unclear, I don't see a documented limit. This might depend on whether you're on the delayed or realtime feed, or it may just be a limit per subscription request. Our library doesn't currently have a way to add or remove subscriptions to an open feed at the moment, but that's something I could play around with to see if breaking the subscriptions up is the trick.
